hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableOutputFormat.java

```diff
@@ -434,4 +477,61 @@ public CarbonLoadModel getLoadModel() {
       return loadModel;
     }
   }
+
+  public static class CarbonMultiRecordWriter
```

Comment:
I think this class can extend the CarbonRecordWriter and override necessary methods

Reply:
only two methods are there and both need override. Doesn't get any reward from doing this.

Reply:
done. overrided